packages/p2p/src/components/modal-manager/modals/order-details-confirm-modal/order-details-confirm-modal.tsx

```diff
@@ -35,7 +37,7 @@ const OrderDetailsConfirmModal = () => {
             <Modal
                 className='order-details-confirm-modal'
                 is_open={is_modal_open}
-                toggleModal={hideModal}
+                toggleModal={() => hideModal()}
```

Comment:
I changed it to an anonymous function as TS was complaining if we pass it as a function reference